graphene_sqlalchemy/tests/test_batching.py

```diff
@@ -531,6 +565,70 @@ def resolve_reporters(self, info):
     assert len(select_statements) == 2
 
 
+@pytest.mark.asyncio
+def test_batch_sorting_with_custom_ormfield(session_factory):
```

Comment:
This works as expected as far as I can tell. You can sort with `FIRSTNAME_*` or `FIRST_NAME_*` which is still automatically created.